aldryn_forms/utils.py

```diff
@@ -61,13 +60,6 @@ def add_form_error(form, message, field=NON_FIELD_ERRORS):
 
 
 def get_form_render_data(form):
-    form_data = []
-    for field in form.visible_fields():
-        value = form.cleaned_data[field.name]
-        if isinstance(value, models.query.QuerySet):
-            value = ', '.join(map(unicode, value))
-        elif value is None:
-            value = '-'
-        name = field.label if field.label else field.name
-        form_data.append((name, value))
-    return form_data
+    for field in form.form_plugin.get_form_fields():
```

Comment:
`form_plugin.get_form_fields()` returns only fields added by the user as plugins. We currently don't have plugins which render hidden fields, and if we would, I think it makes sense to include them.